zoo-redis/custom-entrypoint.sh

```
@@ -0,0 +1,29 @@
+mkdir -p /usr/local/etc/redis
+
+# Dynamically generate Redis configuration and ACL files here, using environment variables
+echo "aclfile /usr/local/etc/redis/custom_aclfile.acl" > /usr/local/etc/redis/redis.conf
+
+# Generate ACL file using environment variables
+if [ -n "${REDIS_USERNAME}" ] && [ -n "${REDIS_PASSWORD}" ]; then
+    echo "user ${REDIS_USERNAME} on allkeys allchannels allcommands >${REDIS_PASSWORD}" > /usr/local/etc/redis/custom_aclfile.acl
+fi
+
+# Configure the default user
+if [ "$(echo ${REDIS_DISABLE_DEFAULT_USER})" == "true" ]; then
```

Shellcheck

[warning] In POSIX sh, == in place of = is undefined.

(SC3014)